force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +315,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        Integer indexWords = -1;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for(String word : words){
+           if(word == wordToFind){
+            indexWords = words.indexOf(wordToFind);
+            break;
+           }
+        }
+        return indexWords; // Replace null with the variable you used to store the result
```

Comment:
great work breaking out of the loop when the match was found. Lost of students miss that. 

Below is my solution is an example of a slightly different technique. I tend to not assign values to variables if I am not going to do anything that that data but return it.  A little more concise, and I don't think any less readable. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```